### PR TITLE
OADP-2205: DOC: OADP features and plugins Page missing VSM plugin

### PR DESCRIPTION
**Pull request by anarnold97**
OADP 1.2; OCP 4.12+

[OADP-2205](https://issues.redhat.com/browse/OADP-2205)

Version(s):

/branch/enterprise-4.12
/branch/enterprise-4.13
/branch/enterprise-4.14

Issue:

[OADP-2205](https://issues.redhat.com/browse/OADP-2205)
Link to docs preview:

[OADP plugins](https://62379--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins.html#oadp-plugins_oadp-features-plugins)
QE review:
- [X ] QE has approved this change. - QE approved (see Jira).
-

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
Please hold this PR and do not merge just yet.

**Review by stillalearner — APPROVED** [review]
lgtm

File: modules/oadp-plugins.adoc [hunk 1]
```diff
@@ -29,10 +29,13 @@ OADP also provides plugins for {product-title} resource backups, OpenShift Virtu
 |`kubevirt` |Backs up and restores OpenShift Virtualization resources. ^[2]^ |Object store
 
 |`csi` |Backs up and restores volumes with CSI snapshots. ^[3]^ |Cloud storage that supports CSI snapshots
+
+|`vsm` |VolumeSnapshotMover relocates snapshots from the cluster into an object store to be used during a restore process to recover stateful applications, in situations such as cluster deletion. ^[4]^ |Object store
```
Comment (adellape) [inline modules/oadp-plugins.adoc:33]: I notice this is the only entry that explains what the plugin value stands for. Also it's a complete sentence, which breaks with the other "Function" descriptions in the table. Not sure if this is too awkward, but for style consistency maybe something like:

> Relocates snapshots from the cluster into an object store to be used during a restore process to recover stateful applications, for example during cluster deletion. Stands for VolumeSnapshotMover.

Comment (anarnold97) [reply modules/oadp-plugins.adoc:33]: @adellape Honestly, I need to change the other sections to provide more information. But this is part of a larger rewrite. Are you ok to accept this until I make a change? I am currently working on [improving the structure](https://github.com/openshift/openshift-docs/pull/62396) of the docs, and then will move to a rewrite
Thanks

**Pull request merged**